### ISSUE TITLE
eval: System Limit 

### ISSUE THREAD

**Issue by jesseshieh** [bitwalker/distillery#619]
Hi! Thanks again for distillery which is amazing. I'm trying to construct an eval command that is very long, but I get the following error. It looks like because it's converting the binary to an atom, it's hitting [the 255 atom limit](http://erlang.org/doc/efficiency_guide/advanced.html#system-limits). I saw that you were able to workaround this in #537, but I didn't see a linked commit or pull request so I'm not sure what the fix was. Is this something that is easy to fix or workaround? 

```
bin/app eval "IO.inspect 1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1"
** (SystemLimitError) a system limit has been reached
    :erlang.binary_to_atom("IO.inspect 1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1+1", :utf8)
    (distillery) lib/entry.ex:115: Mix.Releases.Runtime.Control.parse_args/3
    (distillery) lib/entry.ex:44: Mix.Releases.Runtime.Control.main/1
    (stdlib) erl_eval.erl:677: :erl_eval.do_apply/6
    (elixir) lib/code.ex:232: Code.eval_string/3

```

- What are the expected results? No length limit for eval command.
- What version of Distillery? 2.0.12
- What OS, Erlang/Elixir versions are you seeing this issue on? elixir 1.7.4 erlang 21.0 ubuntu 16.04
- If possible, also provide your `rel/config.exs`, as it is often
  my first troubleshooting question, and you'll save us both time :) the default generated one

**Comment by nulian** [comment]
The same issue with the rpc command.

**Comment by bitwalker** [comment]
This is fixed via `artificery` 0.4.2. You can update with `mix deps.update artificery`
